Company API Flow/Adverse Action of Single Employee.md

```diff
@@ -0,0 +1,38 @@
+# cURL
+
+curl --location --request GET 'https://api.us.springverify.com/company/action/adverse?adverse_action_id=2504d7c7-3f68-47a7-b9dc-be2adb99936e' \
```

Comment:
no need to add https://api.us.springverify.com/ everywhere. It can be there in the introduction